src/controllers/imgurController.js

```diff
@@ -0,0 +1,68 @@
+const { ImgurClient } = require('imgur');
+const User = require('../models/User');
+
+const imgurClient = new ImgurClient({
+  clientId: process.env.IMGUR_CLIENT_ID,
+  clientSecret: process.env.IMGUR_CLIENT_SECRET,
+  refreshToken: process.env.IMGUR_REFRESH_TOKEN,
+});
+
+const renderUploadPage = (req, res) => {
+  res.render('imgur');
```

Comment:
We need to pass the EJS page the userId here:

![](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/0686acf1-9839-416e-aedb-a8eb4f94d4ab)